v23.1/export-data-with-changefeeds.md

```diff
@@ -9,11 +9,12 @@ When you create an {{ site.data.products.enterprise }} changefeed, you can inclu
 
 {% include_cached new-in.html version="v23.1" %} You can also [schedule a changefeed](#create-a-scheduled-changefeed-to-export-filtered-data) to use a changefeed initial scan for exporting data on a regular cadence. 
 
-The benefits of using changefeeds for this use case compared to an export, include:
+The benefits of using changefeeds for this use case compared to an [export](export.html), include:
```

Comment:
I think this would be more emphatic advice with "for this ... instead of ..." (here and in other use)

Reply:
I like that, changed!